e2e/cypress/integration/enterprise/ldap_group/team_and_channel_assign_roles_spec.js

```diff
@@ -53,40 +70,22 @@ describe('System Console', () => {
         cy.findByTestId('search-input').type(`${teamName}{enter}`);
         cy.findByTestId(`${teamName}edit`).click();
 
-        // # Wait until the groups retrieved and show up
-        cy.wait(TIMEOUTS.SMALL);
-
-        // # Remove all existing groups
-        cy.get('#groups-list--body').then((el) => {
-            if (el[0].childNodes[0].innerText !== 'No groups specified yet') {
-                for (let i = 0; i < el[0].childNodes.length; i++) {
-                    cy.get('#group-actions').click();
-                }
-            }
-        });
-
-        // # Save the setting and navigate back to page
-        saveAndNavigateBackTo(teamName);
-
```

Comment:
This failed when no LDAP group is linked.  It happened on initial run. Directly delete via API. Since this is a routine test preparation, makes sense to move into before hook.